src/components/30-organisms/carousel/index.js

```diff
@@ -0,0 +1,228 @@
+import { LitElement, html, css, unsafeCSS } from 'lit-element';
+
+/* eslint-disable import/no-extraneous-dependencies */
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+import Swipe from './swipe';
+import debounce from '../../../utils/debounce';
+
+const ELEMENT_NODE = 1;
+const ANIMATION_LEFT_CLASS = 'animation-left';
+const ANIMATION_RIGHT_CLASS = 'animation-right';
+
+class AXACarousel extends LitElement {
+  static get tagName() {
+    return 'axa-carousel';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      autorotatedisabled: { type: Boolean },
+      autorotatetime: { type: Number },
+      keysenabled: { type: Boolean },
+      _animationWrapperClass: { type: String },
```

Comment:
Why a underscored HTML attribute? if u intzended it as private variable in the class, please add it to the constructor. This code here will enable attr observing.

Reply:
Lit element only binds props to template - class variables not being updated